src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfBus.java

```diff
@@ -367,6 +348,11 @@ public void setCalculatedQ(double calculatedQ) {
         return generators;
     }
 
+    @Override
+    public LfLoads getLfLoads() {
```

Comment:
I am not sure this LfLoads is more performant than List<LfLoad> but why not. Maybe we should create it lazily to only impact LF with constant power factor load distribution.